@navikt/core/react/src/date/datepicker/datepicker.stories.tsx

```diff
@@ -441,33 +422,25 @@ export const ModalDemo = () => {
   });
 
   return (
-    <Modal open header={{ heading: "Modal-demo" }} width="1024px">
+    <Modal open header={{ heading: "Modal-demo" }}>
       <Modal.Body style={{ position: "relative" }}>
-        <BodyLong>
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Maiores nisi
-          incidunt ipsum cupiditate nostrum nesciunt, corrupti nihil at atque
-          animi ab aut. Quam iusto harum eligendi magnam nulla repudiandae
-          molestias.
+        <BodyLong spacing>
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit.
         </BodyLong>
 
-        <HGrid gap="6" columns={{ xs: 1, lg: 2 }}>
-          <HStack justify="center" align="start">
-            <DatePicker.Standalone
-              fromDate={new Date("Aug 23 2019")}
-              toDate={new Date("Feb 23 2024")}
-            />
-          </HStack>
-          <DatePicker {...datepickerProps} dropdownCaption>
-            <DatePicker.Input {...inputProps} label="Velg dato" />
-          </DatePicker>
-          <HStack justify="center">
-            <DatePicker.Standalone
-              fromDate={new Date("Aug 23 2019")}
-              toDate={new Date("Feb 23 2024")}
-            />
-          </HStack>
-        </HGrid>
+        <DatePicker {...datepickerProps} dropdownCaption>
+          <DatePicker.Input
+            {...inputProps}
+            label="Velg dato"
+            description="Format: dd.mm.yyyy"
+          />
+        </DatePicker>
       </Modal.Body>
+      <Modal.Footer>
+        <Button>Neste</Button>
+        <Button variant="secondary">Tilbake</Button>
+        <Button variant="tertiary">Avbryt</Button>
+      </Modal.Footer>
     </Modal>
   );
 };
```

Comment:
Storyen ModalDemo blir blank i snapshot. Tror det bare er å legge til dette:
```suggestion
};
ModalDemo.parameters = { chromatic: { pauseAnimationAtEnd: true } };
```